docs/workflow/requirements/macos-requirements.md

```diff
@@ -23,47 +23,14 @@ Install the following packages:
 - cmake 3.15.5 or newer
 - autoconf
 - automake
+- icu4c
 - libtool
+- openssl 1.1 or newer
 - pkg-config
 - python3
-- icu4c
 
 The lines to install all the packages above using Homebrew.
 
 ```
-brew install cmake autoconf automake libtool pkg-config python3 icu4c
-brew link --force icu4c
-```
-
-OpenSSL
--------
-
-To build the libraries on macOS, you must install and configure links for OpenSSL 1.1.
-
-```sh
-brew install openssl
-
-# You might need to "link" pkg-config:
-brew link pkg-config
-
-# We need to make the runtime libraries discoverable, as well as make
-# pkg-config be able to find the headers and current ABI version.
-#
-# Ensure the paths we will need exist
-mkdir -p /usr/local/lib/pkgconfig
-
-# The rest of these instructions assume a default Homebrew path of
-# `/usr/local/opt/<module>`, with `brew --prefix` returning `/usr/local`
-# and `brew --prefix openssl` returning `/usr/local/opt/openssl@1.1`.
-# In this case, `brew info openssl` shows:
-# `openssl@1.1: stable 1.1.1d (bottled) [keg-only]`.
-
-# Runtime dependencies
-ln -s /usr/local/opt/openssl\@1.1/lib/libcrypto.1.1.dylib /usr/local/lib/
-ln -s /usr/local/opt/openssl\@1.1/lib/libssl.1.1.dylib /usr/local/lib/
-
-# Compile-time dependencies (for pkg-config)
-ln -s /usr/local/opt/openssl\@1.1/lib/pkgconfig/libcrypto.pc /usr/local/lib/pkgconfig/
-ln -s /usr/local/opt/openssl\@1.1/lib/pkgconfig/libssl.pc /usr/local/lib/pkgconfig/
-ln -s /usr/local/opt/openssl\@1.1/lib/pkgconfig/openssl.pc /usr/local/lib/pkgconfig/
+brew install cmake autoconf automake icu4c libtool openssl pkg-config python3
```

Comment:
Do you know if one can run tests locally without the links? (we would not see it the in CI as tests do run on different machine)

Reply:
Assuming Homebrew is still not linking the libraries into /usr/local/lib/, we need to keep the runtime dependencies section.

But as long as src\libraries\Native\build-native.sh is sourcing the script that adds the default homebrew path to the pkg-config probe path, we can remove the pkgconfig-specific steps from the docs.

Reply:
If this is meant for developers, I'm wondering if we can add rpath so runtime would find them as well automatically. Or add note that somebody can set LD_LIBRARY_PATH as alternative to linking. 
In either case I would be nice if one does not have to do any linking and direct file manipulation.